forc/src/ops/forc_init.rs

```diff
@@ -106,82 +74,92 @@ pub fn init(command: InitCommand) -> Result<()> {
+
+    if command.verbose {
+        println_light_blue(&format!(
+            "\nUsing project directory at {}",
+            canonicalize(&project_dir)?
+        ))
+    }
 
     // Make directory for tests
-    fs::create_dir_all(Path::new(&project_name).join("tests"))?;
+    fs::create_dir_all(Path::new(&project_dir).join("tests"))?;
 
     // Insert default manifest file
     match program_type {
         Library => fs::write(
-            Path::new(&project_name).join(constants::MANIFEST_FILE_NAME),
-            defaults::default_manifest(&neat_name, constants::LIB_ENTRY),
+            Path::new(&project_dir).join(constants::MANIFEST_FILE_NAME),
+            defaults::default_manifest(&project_name, constants::LIB_ENTRY),
         )?,
         _ => fs::write(
-            Path::new(&project_name).join(constants::MANIFEST_FILE_NAME),
-            defaults::default_manifest(&neat_name, constants::MAIN_ENTRY),
+            Path::new(&project_dir).join(constants::MANIFEST_FILE_NAME),
+            defaults::default_manifest(&project_name, constants::MAIN_ENTRY),
         )?,
     }
 
     // Insert default test manifest file
     fs::write(
-        Path::new(&project_name).join(constants::TEST_MANIFEST_FILE_NAME),
-        defaults::default_tests_manifest(&neat_name),
+        Path::new(&project_dir).join(constants::TEST_MANIFEST_FILE_NAME),
+        defaults::default_tests_manifest(&project_name),
     )?;
 
-    // Insert src based on program_type
     match program_type {
         Contract => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::MAIN_ENTRY),
             defaults::default_contract(),
         )?,
         Script => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::MAIN_ENTRY),
             defaults::default_script(),
         )?,
         Library => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::LIB_ENTRY),
             defaults::default_library(&project_name),
         )?,
         Predicate => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::MAIN_ENTRY),
             defaults::default_predicate(),
         )?,
     }
 
     // Insert default test function
-    fs::write(
-        Path::new(&project_name).join("tests").join("harness.rs"),
-        defaults::default_test_program(&project_name),
-    )?;
+    let harness_path = Path::new(&project_dir).join("tests").join("harness.rs");
+    fs::write(&harness_path, defaults::default_test_program(&project_name))?;
+
+    if command.verbose {
+        println_light_blue(&format!(
+            "\nCreating test harness at {}",
+            canonicalize(&harness_path)?
+        ));
+    }
```

Comment:
Maybe we should move this earlier to before we write to `harness_path` above?

Reply:
@mitchmindtree Super minor this. I guess I can change to past tense (i.e., `Created`), since I think putting this rather ugly `println_light_blue` in between the `let harness_path` and the`fs::write` makes this a little harder to read

Reply:
Yeah using past tense instead SGTM.